nix/sources.json

```diff
@@ -0,0 +1,14 @@
+{
+    "nixpkgs": {
+        "branch": "nixos-21.05",
+        "description": "Nix packages collection",
+        "homepage": "https://github.com/NixOS/nixpkgs",
+        "owner": "NixOS",
+        "repo": "nixpkgs",
+        "rev": "e85f0175e3effe9ba191d66c09e8f1b7d6362d5e",
+        "sha256": "1qr2hmymbzwj8jrz6smcgc04scgwp2v5070x22k4fvh261g1n4zw",
+        "type": "tarball",
+        "url": "https://github.com/NixOS/nixpkgs/archive/e85f0175e3effe9ba191d66c09e8f1b7d6362d5e.tar.gz",
+        "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
```

Comment:
Is `<foo>` doing string interpolation somehow?

Reply:
Niv uses this template when updating a pin.